deployment-template/ecs/aws-otel-ec2-sidecar-deployment-cfn.yaml

```diff
@@ -131,7 +131,7 @@ Resources:
           Links:
             - aws-collector
         - Name: traffic-generator-xray
-          Image: 'curlimages/curl:latest'
+          Image: 'public.ecr.aws/itx-devops/curl:latest'
```

Comment:
Who owns `itx-devops`?

Reply:
Its a mirror image from the Inbound Transportation Experience team, but didn't have full context over ownership.

Reply:
I do not want to introduce a dependency to this ECR repo. If we require the `curl` image I want to depend on officially support images and tags.

Reply:
The is no available officially supported tags in the Public ECR. I Think this image should be fine as its well used with 50M+ downloads, if not only then only way to use it by mirroring the official image 
![image](https://github.com/aws-observability/aws-otel-collector/assets/41936996/2549e4e6-262f-422e-b8f9-ec373071f28a)